src/project-tests/pomodoro-clock-tests.js

```diff
@@ -78,476 +84,718 @@ export default function createPomodoroClockTests() {
         // start the pomodoro
-        clickButtonsById([_start_stop]);
+        clickButtonsById([_startStop]);
         return new Promise((resolve, reject) => {
-          const timeLeft = document.getElementById("time-left");
-          const observer = observeElement("time-left", modType => {
-            if (timeLeft.innerHTML === "00:00") {
-              // once timer has reached zero wait 1.5 seconds then reset and see if every default value is reset
-              setTimeout(_ => {
+          const timeLeft = document.getElementById('time-left');
+          const observer = observeElement('time-left', () => {
+            if (timeLeft.innerHTML === '00:00') {
+              // once timer has reached zero wait 1.5 seconds then reset and 
+              // see if every default value is reset
+              setTimeout(() => {
                 resetTimer();
-                const timerLabelAfterReset = document.getElementById('timer-label').innerText;
-                const secondsAfterReset = getSeconds(document.getElementById("time-left").innerHTML);
+                const timerLabelAfterReset = document.getElementById(
+                  'timer-label'
+                ).innerText;
+                const secondsAfterReset = getSeconds(
+                  document.getElementById('time-left').innerHTML
+                );
 
                 // see if timer label changed back
                 if (orignalTimerLabel !== timerLabelAfterReset) {
-                  reject(new Error("Default timer label was not properly reset"));
+                  reject(
+                    new Error('Default timer label was not properly reset')
+                  );
                 }
 
-                // wait another 1.5 seconds to be sure value has not changed (pomodoro is stopped)
-                setTimeout(_ => {
-                  const breakLengthAfterResetIsCorrect = document.getElementById("break-length").innerHTML == 5;
-                  const sessionLengthAfterResetIsCorrect = document.getElementById("session-length").innerHTML == 25;
-                  if (!breakLengthAfterResetIsCorrect || !sessionLengthAfterResetIsCorrect) {
-                    reject(new Error("Default values for break length and session length were not properly reset"));
+                // wait another 1.5 seconds to be sure value has not changed 
+                // (pomodoro is stopped)
+                setTimeout(() => {
+                  const breakLenAfterResetCorrect = (
+                    document.getElementById('break-length').innerHTML === '5'
+                  );
+                  const sessionLenAfterResetCorrect = (
+                    document.getElementById('session-length').innerHTML === '25'
+                  );
+                  if (
+                    !breakLenAfterResetCorrect || !sessionLenAfterResetCorrect
+                  ) {
+                    reject(
+                      new Error('Default values for break length and session' +
```

Comment:
If the parameter(s) can't fit on one line, you should put it on its own line. Something like:

```
new Error(
  'Default values for break length and session' +
  'length were not properly reset'
);
```